### DISCUSSION TITLE
Mobile editing

### DISCUSSION THREAD

**Discussion by CuzImBisonratte** [community/community#10616] [Mobile] [answered]
I know it wouldn't be very practical to edit full code files, but i would like it soo much, if i could fix some typos or such things easily, without getting out my laptop every time.

**Comment by taki-on** [comment]
Hi @CuzImBisonratte, thank you for your feedback! Currently the repo files are read-only on mobile, but we have some early stage discussions to support editing those files on mobile. Please stay tuned!

**Reply by T-Felm-Lee** [reply]
Would also love to see this.  Fixing small mistakes would make me not leave the app at all.  

Currently I will read my code on the app because it's very readable and easy to navigate then either create an issue for a simple typo, or use the browser mobile website to edit.

**Reply by davidd1001** [reply]
It sounds like you're expressing the desire for a more seamless and convenient experience when it comes to editing code within a mobile app. While I don't have real-time information on specific app features or updates, I can provide some general suggestions that could improve the user experience for editing code on a mobile app: You can also use[ ai tools](www.advancewithai.net)

In-App Editing Features: Consider suggesting to the app developers the inclusion of basic code editing features directly within the mobile app. This could include functionalities like syntax highlighting, code completion, and easy navigation through the codebase.

Quick Edit Mode: An efficient "Quick Edit" mode that allows users to make small changes or fixes without navigating away from the code view could be beneficial. This could involve a pop-up editor or a similar interface that facilitates quick edits.

Integrated Feedback Mechanism: Instead of creating issues externally, an integrated feedback mechanism within the app could streamline the process of reporting and fixing small mistakes. This could include a button for reporting issues or suggesting corrections directly from the code view.

Collaborative Editing: If the app involves collaborative coding or version control, enhancing mobile support for these features would allow users to contribute to codebases more effectively from their mobile devices.

Offline Editing Capability: Consider suggesting the ability to edit code offline and sync changes when a stable internet connection is available. This can be useful for users who want to make quick edits on the go without relying on a continuous internet connection.

Enhanced Mobile Website Editing: If you're using the browser mobile website for editing, suggest improvements for a smoother editing experience. This could include an optimized mobile code editor, better navigation, and an improved user interface.

Remember to provide specific examples and use cases when suggesting these features to make your feedback more actionable for the developers. Additionally, check for updates from the app developers or community forums to see if any of these features have been introduced in recent releases.

**Comment by Selena20191** [comment]
> I know it wouldn't be very practical to edit full code files, but i would like it soo much, if i could fix some typos or such things easily, without getting out my laptop every time.

👍

**Comment by spaltrowitz** [comment, marked as answer]
thanks for your suggestion @CuzImBisonratte! this has now shipped https://github.blog/changelog/2022-11-15-github-for-ios-edit-files-from-browse-code

**Reply by CuzImBisonratte** [reply]
Thank all of you so much for this feature 👍, i have already used it today without remembering i asked for it haha.
